### PR TITLE
new limit - auto agg disables dataset download

### PR DESCRIPTION
**Pull request by lesliemui**
per https://docs.microsoft.com/en-us/power-bi/enterprise/aggregations-auto#considerations-and-limitations
turning on automatic aggregations disables dataset .pbix download

### CODE REVIEW

**Comment by PRMerger16** [comment]
@lesliemui : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by maggiesMSFT — APPROVED** [review]

**Comment by maggiesMSFT** [comment]
#sign-off

File: powerbi-docs/create-reports/service-export-to-pbix.md [hunk 1]
```diff
@@ -97,6 +97,7 @@ This section lists the limitations of downloading a .pbix file. This section is
 * Datasets modified by using the [XMLA endpoint](../enterprise/service-premium-connect-tools.md) can't be downloaded to a .pbix file.
 * You can't download datasets based on data imported from Onedrive for Business.
 * Downloading a .pbix file that is larger than 1 GB isn't supported.
+* Datasets with [automatic aggregations](../enterprise/aggregations-auto#considerations-and-limitations) enabled can't be downloaded to a .pbix file
```
Comment (Court72) [inline powerbi-docs/create-reports/service-export-to-pbix.md:100]:
```suggestion
* Datasets with [automatic aggregations](../enterprise/aggregations-auto.md#considerations-and-limitations) enabled can't be downloaded to a .pbix file.
```

**Comment by PRMerger7** [comment]
@Court72 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**